engine/planner.go

```diff
@@ -151,7 +153,7 @@ func (p *planner) mark(ctx context.Context, n int) error {
 		logger.WithField("servers-to-terminate", n).
 			WithField("servers-running", len(servers)).
 			WithField("min-pool", p.min).
-			Debugf("abort terminating %d instances to ensure minimum capacity met")
+			Debugf("abort terminating %d instances to ensure minimum capacity met", n)
```

Comment:
As it already got the field `servers-to-terminate` I would just rephrase the message and remove the parameter.

Reply:
Thanks. I've just updated.